vcsutils/utils.go

```diff
@@ -224,3 +231,15 @@ func unzipFile(f *zip.File, destination string) (err error) {
 	}()
 	return safeCopy(destinationFile, zippedFile)
 }
+
+func CreateDotGitFolderWithRemote(path, remoteName, remoteUrl string) error {
```

Comment:
Let's stick to the convention to put documentation above all public functions
```suggestion
// CreateDotGitFolderWithRemote ...
func CreateDotGitFolderWithRemote(path, remoteName, remoteUrl string) error {
```

Reply:
done